docs/docs/cmd/search/externalconnection/externalconnection-get.md

```diff
@@ -0,0 +1,37 @@
+# search externalconnection get
+
+Retrieves an external connection for Microsoft Search by Id or Name.
+
+## Usage
+
+```sh
+m365 search externalconnection get [options]
+```
+
+## Options
+
+`-i, --id [id]`
+: Developer-provided unique ID of the connection within the Azure Active Directory tenant
```

Comment:
Let's adjust this a bit so that it's more relevant in the context of the `get` command. The way it sounds now, it's more like something we'd say in context of an `add` command

```suggestion
: ID of the connection to retrieve. Specify either `id` or `name` but not both
``